scripts/armv7a9-zynq7000.sh

```diff
@@ -1,20 +1,29 @@
 #!/bin/bash
 #
-# Shell script for running phoenix-rtos loader on QEMU (zynq-7000)
+# Shell script for running phoenix-rtos loader on Qemu fork from Xilinx (zynq-7000)
 #
 # Copyright 2021 Phoenix Systems
 # Author: Hubert Buczynski
 #
 
-IMG_ZYNQ7000="$(dirname "${BASH_SOURCE[0]}")/../_boot/plo-ram-armv7a9-zynq7000.img"
+IMG_PLO_ZYNQ7000="$(dirname "${BASH_SOURCE[0]}")/../_boot/plo-armv7a9-zynq7000-qemu.img"
+IMG_FLASH_QEMU="$(dirname "${BASH_SOURCE[0]}")/../_build/armv7a9-zynq7000/prog/flash-armv7a9-zynq7000.bin"
+IMG_PHOENIX_ZYNQ7000="$(dirname "${BASH_SOURCE[0]}")/../_boot/phoenix-armv7a9-zynq7000.disk"
+DTB_ZYNQ7000="$(dirname "${BASH_SOURCE[0]}")/../scripts/zynq7000-zc702.dtb"
 
-if [ ! -f "$IMG_ZYNQ7000" ]; then
-	echo "File $IMG_ZYNQ7000 does not exist"
+if [[ ! -f "$IMG_PLO_ZYNQ7000" && ! -f "$IMG_PHOENIX_ZYNQ7000" && ! -f "$DTB_ZYNQ7000" ]]; then
+	echo "Missing required files, check $IMG_PLO_ZYNQ7000, $IMG_PHOENIX_ZYNQ7000 and $DTB_ZYNQ7000"
 	exit 1
 fi
 
-exec qemu-system-arm \
-	-M xilinx-zynq-a9 \
+# Create file as counterpart of NOR flash memory
+dd if=/dev/zero of="$IMG_FLASH_QEMU" bs=16M count=1
+dd if="$IMG_PHOENIX_ZYNQ7000" of="$IMG_FLASH_QEMU" bs=1 seek=0 conv=notrunc
```

Comment:
```suggestion
rm -f "$IMG_FLASH_QEMU"
dd if="$IMG_PHOENIX_ZYNQ7000" of="$IMG_FLASH_QEMU" bs=4M
truncate -s 16M "$IMG_FLASH_QEMU"
```
bs=1 is somewhat slow, truncate is simpler that writing zeroes via dd, also makes these zeroes virtual, so they don't really take disc space.

Reply:
Thanks. Fixed